### ISSUE TITLE
Task #13 [Stealthy Operation]: How to suppress powershell output streams (XML)?

### ISSUE THREAD

**Issue by deetee1** [mitre/training#133]
I'm doing Task #13 which uses the Hunter adversary profile with base64 obfuscation and 10/20 seconds jitter, with 1 agent (Windows 10 VM, which is part of a domain). I noticed that the output of the base64-encoded powershell commands return XML streams.

e.g. for "Identify local users", this is the output of `powershell -Enc RwBlAHQALQBXAG0AaQBPAGIAagBlAGMAdAAgAC0AQwBsAGEAcwBzACAAVwBpAG4AMwAyAF8AVQBzAGUAcgBBAGMAYwBvAHUAbgB0AA==`:
```
#< CLIXML
<Objs Version="1.1.0.1" xmlns="http://schemas.microsoft.com/powershell/2004/04"><Obj S="progress" RefId="0"><TN RefId="0"><T>System.Management.Automation.PSCustomObject</T><T>System.Object</T></TN><MS><I64 N="SourceId">1</I64><PR N="Record"><AV>Preparing modules for first use.</AV><AI>0</AI><Nil /><PI>-1</PI><PC>-1</PC><T>Completed</T><SR>-1</SR><SD> </SD></PR></MS></Obj></Objs>
```
![image](https://user-images.githubusercontent.com/62365248/130752599-cba5cf72-869b-4651-aa00-4ca640b27089.png)
![image](https://user-images.githubusercontent.com/62365248/130751939-2bbcf1e1-fe5c-4739-94d5-835fe800d834.png)

When I run the same powershell command directly on the command line in the Win10 VM, the output comes out fine:
![image](https://user-images.githubusercontent.com/62365248/130752265-8a5066cb-c94f-4825-afd2-95c18e6fa4b3.png)

How do we go about getting the correct output in CALDERA instead of the XML streams?

**Comment by uruwhy** [comment]
@deetee1 What version of caldera are you using? Does this occur with all base64-encoded commands? Does this happen with any normal powershell abilities (non-enoded)?

**Comment by deetee1** [comment]
@uruwhy I'm using v3.1.0 with docker. 

No, this occurred for the following base64-encoded commands, please see below:

Find user processes
![image](https://user-images.githubusercontent.com/62365248/132356091-9c54c928-eafc-4d79-8bdc-cea7c88575ab.png)

View admin shares
![image](https://user-images.githubusercontent.com/62365248/132356133-2e4e54e4-5c56-4928-ad08-8c1bdc0a0635.png)

Identify firewalls
![image](https://user-images.githubusercontent.com/62365248/132356349-67a33a91-cec3-4c2d-ac18-c2beb9ff38b1.png)

Create staging directory
![image](https://user-images.githubusercontent.com/62365248/132356564-c403dde8-15fb-467f-a3b5-cc3df4579201.png)

Stage sensitive files
![image](https://user-images.githubusercontent.com/62365248/132356648-d74a9da9-9ebf-4b90-affe-356e43d5c1e9.png)

This does not happen with non-encoded powershell commands.

**Comment by uruwhy** [comment]
I was able to reproduce the same results as you - it looks like not every base64 encoded powershell command is affected, but the majority. I'll do some digging to see what's going on. Thank you for bringing this up!

**Comment by uruwhy** [comment]
So I came across an interesting finding... I tried replicating this on a domain user account and on a local user account (both on a workstation that's part of an AD domain) - I don't get the XML output when running the operation on my local user account. If my workstation is connected to the domain controller, then I don't get the XML output either. I only get the XML output when running the operation on a workstation when the domain controller is offline - maybe there's something going on with trying to connect to the DC in the background when running as a domain user, hence we get the XML as stderr output. Was there an active DC connected to your domain when you tried this on your workstation?
